InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/DriveSubsystem.java

```diff
@@ -99,24 +111,27 @@ public DriveSubsystem(Config config, NavigationSubsystem navigationSubsystem, OI
 
         DifferentialDriveKinematicsConstraint kinematicsConstraint = new DifferentialDriveKinematicsConstraint(
             DRIVE_UTILS.KINEMATICS,
-            config.drive.maxAllowedSpeed_ips * DriveConstants.METERS_PER_INCH
+            config.auto.cruiseSpeed_mps * DriveConstants.METERS_PER_INCH
         );
 
         TrajectoryConfig trajectoryConfig = new TrajectoryConfig(
-            config.drive.maxAllowedSpeed_ips * DriveConstants.METERS_PER_INCH,
-            DRIVE_UTILS.MAX_ACCELERATION_MPSPS
+            config.auto.cruiseSpeed_mps,
+            config.auto.maxAcceleration_mps
         );
         trajectoryConfig.addConstraint(voltageConstraint);
         trajectoryConfig.addConstraint(kinematicsConstraint);
 
         autoTrajectory = TrajectoryGenerator.generateTrajectory(
-            new Pose2d(new Translation2d(0, 0), Rotation2d.fromDegrees(0)),
-            List.of(new Translation2d(1, 1), new Translation2d(2, -1)),//FieldConstants.OUR_POWER_CELL_1, FieldConstants.OUR_POWER_CELL_2),
-            new Pose2d(new Translation2d(3, 0), Rotation2d.fromDegrees(0)),
+            new Pose2d(FieldConstants.FRONT_OF_POWER_PORT, Rotation2d.fromDegrees(90)),
```

Comment:
If we start at -90 we don’t want to rotate to 90. That would put our intake the wrong direction.

Reply:
The NavX has angles backward so I offset it by -90 then when reading the value negate it again, so WPI code should already think its at 90